function_maps.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"math/rand"
+	"text/template"
+	"time"
+
+	sprig "github.com/go-task/slim-sprig"
```

Comment:
Is there a reason to prefer slim-sprig over sprig in our case?

Reply:
Also, are there any Helm-specific functions that might be useful for us to import or duplicate?

https://helm.sh/docs/chart_template_guide/function_list/